index.bs

```diff
@@ -85,8 +88,8 @@ The DeviceOrientationSensor Interface {#deviceorientationsensor-interface}
 <pre class="idl">
   [Constructor(optional SensorOptions sensorOptions)]
   interface DeviceOrientationSensor : Sensor {
-    void populateQuaternion(Float32Array array);
-    void populateMatrix(Float32Array array);
+    void populateQuaternion(Float32Array buffer);
```

Comment:
But when this data is buffered, and you get > 1 sample delivered on the animation frame, you'll need a different timestamp for each of the buffered data.

So what's the plan to handle that?

Are you suggesting different TypeArray organisations depending on whether the data is buffered or not? That seems like a rather bad idea.